packages/server-admin-ui/src/views/appstore/Apps/AppsList.js

```diff
@@ -65,6 +76,8 @@ function mainIcon (app) {
 }
 
 function installApp (name, version) {
+  this.state.installing[name] = true
+  this.setState(this.state)
```

Comment:
The proper way here is to pass setState a function (state, props) => {...}  see https://www.google.com/amp/s/www.freecodecamp.org/news/functional-setstate-is-the-future-of-react-374f30401b6b/amp/